src/Config.ts

```diff
@@ -614,6 +615,11 @@ export class Config implements ConfigParams, ParserConfig {
       {value: this.functionArgSeparator, name: 'functionArgSeparator'},
       {value: this.thousandSeparator, name: 'thousandSeparator'}
     )
+
+    if (GPU === undefined) {
+      this.gpuMode = 'fallback'
+      console.warn('GPU.js not detected. Matrix functions will use plain CPU implementation.')
```

Comment:
@voodoo11 I doubt it should be printed in the console. That's not an error, rather one of two available modes in which the engine works. So far we logged only issues with the config (deprecated options) and invalid license key.